tests/integration/test_logs.py

```diff
@@ -83,3 +92,45 @@ def test_list_tags_log_group(self):
         self.logs_client.delete_log_group(
             logGroupName=group
         )
+
+    def test_put_subscribe_policy(self):
+        iam_client = aws_stack.connect_to_service('iam')
+        lambda_client = aws_stack.connect_to_service('lambda')
+        log_client = aws_stack.connect_to_service('logs')
+
+        role_name = 'role-{}'.format(short_uid())
+        assume_policy_document = {'Version': '2012-10-17',
+                                  'Statement': [
+                                      {
+                                          'Action': 'sts:AssumeRole',
+                                          'Principal': {
+                                              'Service': 'lambda.amazonaws.com'
+                                          },
+                                          'Effect': 'Allow',
+                                          'Sid': '',
+                                      }
+                                  ]
+                                  }
+
+        iam_client.create_role(
+            RoleName=role_name,
+            AssumeRolePolicyDocument=json.dumps(assume_policy_document)
+        )
+
+        testutil.create_lambda_function(
+            handler_file=TEST_LAMBDA_PYTHON3, libs=TEST_LAMBDA_LIBS,
+            func_name=TEST_LAMBDA_NAME_PY3, runtime=LAMBDA_RUNTIME_PYTHON36)
+
+        lambda_client.invoke(
+            FunctionName=TEST_LAMBDA_NAME_PY3, Payload=b'{}')
+
+        log_group_name = '/aws/lambda/{}'.format(TEST_LAMBDA_NAME_PY3)
+        log_client.put_subscription_filter(
+            logGroupName=log_group_name,
+            filterName='test',
+            filterPattern='',
+            destinationArn=func_arn(TEST_LAMBDA_NAME_PY3),
+        )
+
+        resp2 = log_client.describe_subscription_filters(logGroupName=log_group_name)
+        self.assertEqual(len(resp2['subscriptionFilters']), 1)
```

Comment:
Can we add an assertion at the end that the destination Lambda has actually been invoked?

Something like this should probably work (ideally wrapped in a `retry` function) - we can import `get_lambda_log_events(..)` from `test_lambda.py` here (or potentially move it as a shared util to `testutils.py`):
```
# assert that the destination Lambda was invoked 1 time
events = get_lambda_log_events(TEST_LAMBDA_NAME_PY3)
self.assertEqual(len(events[0]['Records']), 1)
```